packages/expo/scripts/autolinking.rb

```diff
@@ -24,3 +25,24 @@ def expo_patch_react_imports!(installer, options = {})
 
   Expo::ReactImportPatcher.new(installer, options).run!
 end
+
+def expo_get_config(field, options = {})
+  project_root = options[:project_root] || '..'
+  node_statement = "require('@expo/config').getConfig('#{project_root}').exp.#{field}"
```

Comment:
I wonder if this will get into areas where the config cannot be found or is malformed.

For instance, in the following case, we'd need to ensure the node modules are all installed before we can build:
`app.json`
```json
{
  "expo": {
    "plugins": ["react-native-firebase"]
  }
}
```

Granted this is a minor concern since autolinking has the same constraint.

Reply:
this is true and that's why i tried to add error log when there're execution failures.